### PR TITLE
feat(derek): New nascar river

### PR DESCRIPTION
**Pull request by amberRrucker**
RSWEB-9541
Should not be merged until announcement goes out.

Creating a new nascar river to mirror design linked in the ticket. Only weirdness is that I had to create an override for super small screens to go full width. I also took out legacy example code on the homepage that is no longer there so we can have a proper stakeholder review.

This can be seen here:
http://localhost:3000/derek/incubation/homepage
beneath the announcement section (which isn't the final version but I needed to see it on page)

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
image

Doesn't seem to line up.

**Comment by amberRrucker** [comment]
@WidgetsBurritos I fixed the alignment issue and this is ready for your review.

**Comment by amberRrucker** [comment]
@WidgetsBurritos I split out the new banner work into a separate PR so this one can be reviewed. It makes some adjustments to the homepage banner but it doesn't radically change the banners. Just cleans them up.

**Comment by WidgetsBurritos** [comment]
Should the text align with the section above it at 900px? 

image

**Comment by amberRrucker** [comment]
@WidgetsBurritos I addressed your alignment feedback, thank you. I can't reassign this to you until jira is working again but it's ready.

File: styleguide/_layouts/default.jade [hunk 1]
```diff
@@ -95,5 +95,6 @@ html(lang="en")
   script.
     $(".imacBanner-headline").fitText(1.1, { maxFontSize: '45px' });
   script.
-    $(".refresh-headline").fitText(0.75, { maxFontSize: '55px' });
-    $(".refresh-headline").not('.refresh-headlineHomepage').fitText(0.7, { maxFontSize: '55px' });
+    $(".refresh-headlineHomepage").fitText(0.8, { minFontSize: '25px', maxFontSize: '55px' });
+    $(".refresh-headlineSecond").fitText(0.75, { minFontSize: '25px', maxFontSize: '55px' });
+    $(".refresh-headlineInterior").fitText(0.8, { minFontSize: '25px', maxFontSize: '55px' });
```
Comment (WidgetsBurritos) [inline styleguide/_layouts/default.jade]: I thought you were removing banner changes from this PR?  Requiring new classes for banners and trying to conditionally load javascript is gonna require additional www work. 

This PR should literally only be the bare minimal needed for adding the nascar river.

File: styleguide/_themes/derek/scss/snowflakes/homepage.scss [hunk 2]
```diff
@@ -302,136 +390,3 @@
...
-  padding: $half-padding-full;
-  text-align: right;
-}
-
-.logoShowcase-image {
-  max-height: 40px;
-  max-width: 75%;
-  text-align: center;
-  vertical-align: middle;
-}
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/snowflakes/homepage.scss]: We shouldn't be deleting any CSS or changing existing css.  That doesn't follow the deprecation model we've talked about.  We should create new classes for new elements in the short term and them remove deprecated code at a later time once we're done with it.

File: styleguide/_layouts/default.jade (hunk 1 above)
Comment (amberRrucker) [reply styleguide/_layouts/default.jade]: This PR had to also adjust for styling when "putting it all together" as we discussed previously. It's why we waiting on this one until we merged everything else. I removed the changes to add a new way to do banners. I still had to make style adjustments to make the banners match the mock.

File: styleguide/_themes/derek/scss/snowflakes/homepage.scss (hunk 2 above)
Comment (amberRrucker) [reply styleguide/_themes/derek/scss/snowflakes/homepage.scss]: This code has long been removed from the homepage. It is not in use anywhere on the site and is just adding clutter to the file. No homepage elements will need to be deprecated in order for this to go out. It's simply removing some unused clutter.

Comment (WidgetsBurritos) [reply styleguide/_themes/derek/scss/snowflakes/homepage.scss]: Either way, code removals usually involve extended QE and it's much easier to confirm from a QE perspective when it's not combined with other changes.

**Pull request closed without merging**